erpnext/stock/doctype/item/item.py

```diff
@@ -889,6 +890,11 @@ def validate_auto_reorder_enabled_in_stock_settings(self):
 			if not enabled:
 				frappe.msgprint(msg=_("You have to enable auto re-order in Stock Settings to maintain re-order levels."), title=_("Enable Auto Re-Order"), indicator="orange")
 
+	def validate_valuation_method(self):
+			if not self.valuation_method:
+				method = frappe.db.get_single_value("Stock Settings", "valuation_method") or "FIFO"
+				self.valuation_method = method
```

Comment:
- Set the default in Stock Settings
- Before saving stock settings, if empty set FIFO in there